R/crunchy.R

```diff
@@ -0,0 +1,23 @@
+#' @importFrom shiny div includeCSS includeScript tags
+.onLoad <- function (lib, pkgname="crunchy") {
+    suppressMessages(trace("bootstrapPage", where=shiny::fillPage, tracer=quote({
+        tagList <- function (...) {
+            shiny::tagList(
+                ...,
+                # Load our assets
+                tags$head(
+                    tags$link(rel="stylesheet", type="text/css",
+                        href="https://app.crunch.io/styles.css"),
+                    includeCSS(system.file("extra.css", package="crunchy")),
+                    includeScript(system.file("extra.js", package="crunchy"))
+                ),
+                # Add a placeholder for our auth
+                div(class = "form-group shiny-input-container",
+                    style = "display: none;",
+                    tags$input(id = "token", type = "text", class = "form-control", value = "")
+                )
```

Comment:
The order of the tags in `tagList` doesn't matter here does it? It's not an issue that our auth is at the end instead of the beginning like it used to be?

Reply:
Fair question. TBH I haven't tried serving a crunchy app yet, just got the tests to pass. We should check it out. 

Here's the function I'm injecting into: https://github.com/rstudio/shiny/blob/master/R/bootstrap.R#L29-L47

If we put our stuff first, the loading of CSS might not be in the right order.